src/components/tree/components/treetable.tsx

```diff
@@ -91,6 +96,75 @@ const BodyRow = React.forwardRef<HTMLDivElement, BodyRowProps>((props, ref) => {
     );
 });
 
+
+interface BodyCellProps extends React.HTMLAttributes<HTMLDivElement> {
+    resizeable?: boolean;
+    maxWidth?: number;
+    onDidColumnResize: (event: MouseEvent | TouchEvent, width: number) => void;
+}
+
+const BodyCell = React.forwardRef<HTMLDivElement, BodyCellProps>((props, ref) => {
+    const { resizeable, onDidColumnResize, maxWidth, className, onResize, style, ...rest } = props;
+    const [width, setWidth] = useState<string | number | undefined>(props.style?.width);
+
+    useEffect(() => {
+        if (resizeable) {
+            setWidth(props.style?.width);
+        }
+    }, [props.style?.width]);
+
+    const cell = <div
+        ref={ref}
+        style={{ minWidth: '50px', ...style }}
+        {...rest}
+        className={className}
+        onResize={onResize}
+    />;
+
+    if (!resizeable) {
+        return cell;
+    }
+
+    return (
+        <Resizable
+            minWidth={50}
+            maxWidth={maxWidth}
+            className={classNames('ant-table-cell-resizable', className ?? '')}
+            size={{
+                width,
+            }}
+            onResizeStart={(_event, _direction, ref) => {
+                const row = ref.closest('.ant-table-row');
+                row?.classList.add('ant-table-row-resizing');
```

Comment:
Can `row` be `undefined` here? What would happen if?

Reply:
Theoretically yes as the `closest` method searches for `.ant-table-row` parent, and it *may* not find it, but practically **it will not fail**.
If it is `undefined` , then we will not add the class. The class is used to hide the actions column.